pillow_svg/SvgImagePlugin.py

```diff
@@ -560,8 +560,7 @@ def draw_svg_file(file: str, imagedraw: ImageDraw.ImageDraw, imagesize: Tuple[in
 
 
 def _accept(prefix):
-    prefixStr = prefix.decode("utf-8")
```

Comment:
* `decode` was raising `UnicodeDecodeError` on input like `b'\x89PNG\r\n\x1a\n\x00\x00\x00\rIHDR'`
* `<svg ` - space was removed to allow `<svg/n`